src/amm.rs

```diff
@@ -107,7 +107,7 @@ pub fn calculate_in_given_out(out_reserve: Balance, in_reserve: Balance, amount_
     let buy_price_hp = numerator.checked_div(denominator).ok_or(Overflow)?;
 
     let result = to_balance!(buy_price_hp).ok();
-    round_up!(result.unwrap())
```

Comment:
My bad, overlooked this one.
I can undo the docs but should i undo this?

Reply:
this can be done like this :

let result = to_balance!(sale_price_hp)?;
round_up!(result)